### PR TITLE
Manual fixes: pdf and web version

### PR DESCRIPTION
**Pull request by carlos-encs**

### CODE REVIEW

File: doc/scheduler-job-examples.tex [hunk 1]
```diff
@@ -20,6 +20,7 @@ \subsection{Example Job Script: Fluent}
 \texttt{slurm-249.out}), and/or by examining the ``moment'' file in 
 \texttt{/disk/nobackup/<yourjob>} (hint: it starts with your job-ID) on the node running
 the job. \textbf{Caveat:} take care with journal-file file paths.
+\newpage
```
Comment (smokhov) [inline doc/scheduler-job-examples.tex:23]: Please let LaTeX manage new pages. It's OK the figures float; they aren't too far.

File: doc/scheduler-job-examples.tex [hunk 2]
```diff
@@ -1,7 +1,7 @@
 % ------------------------------------------------------------------------------
 \subsection{Example Job Script: Fluent}
 
-\begin{figure}[htpb]
+\begin{figure}[h!]
```
Comment (smokhov) [inline doc/scheduler-job-examples.tex:4]: Why this change? htpb is generally good and sufficient for this.

File: doc/scheduler-scripting.tex [hunk 3]
```diff
@@ -563,9 +563,10 @@ \subsubsection{Graphical Applications}
 \end{verbatim}
 \normalsize
 
-\begin{figure}[htpb]
-	\includegraphics[width=\columnwidth]{images/pycharm}
-    \caption{PyCharm Starting up on a Speed Node}
+\begin{figure}[h]
+	\centering
+		\includegraphics[width=0.85\textwidth]{images/pycharm}
```
Comment (smokhov) [inline doc/scheduler-scripting.tex:568]: I  suggest keeping `\columnwidth` the way it was.

File: doc/scheduler-scripting.tex [hunk 4]
```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
 
 \end{enumerate}
 
-\begin{figure}[htbp]
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty1}}
+		\includegraphics[scale=0.55]{images/putty1}
 	\caption{SSH tunnel configuration 1}
```
Comment (smokhov) [inline doc/scheduler-scripting.tex:628]: I think `scale` is the problem for HTML render. Base it t on the `\columnwidth` or `\textwidth` instead.

File: doc/scheduler-scripting.tex [hunk 5]
```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
 
 \end{enumerate}
 
-\begin{figure}[htbp]
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty1}}
```
Comment (smokhov) [inline doc/scheduler-scripting.tex:626]: Please keep `\fbox`, it gives good frame around the white spots of the image.

File: doc/scheduler-scripting.tex [hunk 6]
```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
...
 	\caption{SSH tunnel configuration 1}
 	\label{fig:putty1}
 \end{figure}
 
-\begin{figure}[htbp]
+
+
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty2}}
```
Comment (smokhov) [inline doc/scheduler-scripting.tex:633]: Please keep `\fbox`

File: doc/scheduler-scripting.tex [hunk 7]
```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
...
+		\includegraphics[scale=0.90]{images/putty2}
```
Comment (smokhov) [inline doc/scheduler-scripting.tex:636]: Same I think scale is an issue for HTML render

**Review by smokhov — CHANGES_REQUESTED** [review]

File: doc/scheduler-job-examples.tex (hunk 1 above)
Comment (carlos-encs) [reply doc/scheduler-job-examples.tex]: fixed

File: doc/scheduler-job-examples.tex (hunk 2 above)
Comment (carlos-encs) [reply doc/scheduler-job-examples.tex]: fixed

File: doc/scheduler-scripting.tex (hunk 3 above)
Comment (carlos-encs) [reply doc/scheduler-scripting.tex]: fixed

File: doc/scheduler-scripting.tex (hunk 4 above)
Comment (carlos-encs) [reply doc/scheduler-scripting.tex:627]: fixed

File: doc/scheduler-scripting.tex (hunk 5 above)
Comment (carlos-encs) [reply doc/scheduler-scripting.tex:626]: fixed

File: doc/scheduler-scripting.tex (hunk 7 above)
Comment (carlos-encs) [reply doc/scheduler-scripting.tex]: fixed

**Review by smokhov — APPROVED** [review]
